language_detector/main.py

```diff
@@ -4,5 +4,20 @@
 
 def detect_language(text, languages):
     """Returns the detected language of given text."""
-    # implement your solution here
-    pass
+
+    results = {}        
+    
+    for lang in languages:
+        word_count = 0
+        
+        for word in text.split(' '):
+            if word in lang['common_words']:
+                word_count += 1
+                
+        results[lang['name']] = word_count
+    
+    max_count = max(results.values())
+    
+    for lang, count in results.items():
```

Comment:
the max() function followed by this seems like an unnecessary step, have a look at how I did this part by just iterating over the items and saving it whenever its larger than the previous largest one.